tutorials/zabbix-monitoring/index.mdx

```diff
@@ -14,45 +14,50 @@ dates:
   posted: 2019-06-17
 ---
 
-[Zabbix](https://www.zabbix.com/) is an open-source software that offers real-time monitoring of servers, virtual machines, network devices, and web applications. Zabbix provides monitoring metrics network utilization, CPU load and disk space consumption. These metrics are collected and stored in a database.
+Zabbix is a powerful open-source software that offers real-time monitoring for servers, virtual machines, network devices, and web applications. Monitoring metrics such as network utilization, CPU load, and disk space consumption are collected by Zabbix and then stored in a database.
 
-Zabbix allows you to determine the health status of an IT infrastructure, helps you analyze data over time which can be used to plan upgrades in the infrastructure when requirements grow.
+This monitoring tool enables users to determine the health status of their IT infrastructure and analyze data over time. The insights provided by Zabbix can be used to plan upgrades to the infrastructure when requirements grow.
 
-Metrics are collected and stored in a database, allowing to determine the health status of an IT infrastructure, help to analyze data over time which can be used to plan upgrades in the infrastructure when requirements grow.
+In this tutorial, we will be using two Scaleway Production-Optimized [Ubuntu Jammy Jellyfish (22.04 LTS)](https://www.scaleway.com/en/imagehub/) Instances. One Instance will act as a server hosting the Zabbix application, while the other will act as a client being monitored. By following this tutorial, you will learn how to set up Zabbix and take control of your IT infrastructure.
```

Comment:
I think you could put the link for "production-optimized" instead of the one for the imagehub